.github/workflows/c3-e2e.yml

```diff
@@ -1,5 +1,6 @@
-name: C3 E2E Tests
+name: E2E
```

Comment:
This is what shows up in the github UI is it not? I think including C3 in the name would make it much more clear 🤔

Reply:
This will show up `E2E / C3`

Reply:
ah ok 🙂